src/main/java/teammates/ui/webapi/action/RemindFeedbackSessionSubmissionAction.java

```diff
@@ -40,8 +40,8 @@ public ActionResult execute() {
                     + "as the feedback session is not open for submissions.", HttpStatus.SC_BAD_REQUEST);
         }
 
-        FeedbackSessionStudentRemindRequest remindRequest =
```

Comment:
Just checked. We need to add a backend test for instructors (currently only have one for students)